Author: Luca Boccassi <bluca@debian.org>
Forwarded: yes
Description: test: fix D-Bus policy override for TEST-73-LOCALE
 We don't need to allow non-root, and the policy needs to specify destination
 and interface too, to narrow it down
--- a/test/units/TEST-73-LOCALE.sh
+++ b/test/units/TEST-73-LOCALE.sh
@@ -666,14 +666,9 @@
         "https://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
 <busconfig>
         <policy user="root">
-                <allow send_member="SetLocale"/>
-                <allow send_member="SetVConsoleKeyboard"/>
-                <allow send_member="SetX11Keyboard"/>
-        </policy>
-        <policy context="default">
-                <allow send_member="SetLocale"/>
-                <allow send_member="SetVConsoleKeyboard"/>
-                <allow send_member="SetX11Keyboard"/>
+                <allow send_destination="org.freedesktop.locale1" send_interface="org.freedesktop.locale1" send_member="SetLocale"/>
+                <allow send_destination="org.freedesktop.locale1" send_interface="org.freedesktop.locale1" send_member="SetVConsoleKeyboard"/>
+                <allow send_destination="org.freedesktop.locale1" send_interface="org.freedesktop.locale1" send_member="SetX11Keyboard"/>
         </policy>
 </busconfig>
 EOF
